faces-tck/pom.xml

```
@@ -40,8 +40,8 @@
         <maven.settings.xml>${user.home}/.m2/settings.xml</maven.settings.xml>
         <maven.exec.params>-B -e -s ${maven.settings.xml} -f ${tck.root}/faces-tck-${faces.version}/tck/pom.xml</maven.exec.params>
         <maven.exec.projects></maven.exec.projects>
-        <maven.exec.goals>clean install</maven.exec.goals>
         <maven.exec.basic.options>-Ppayara-ci-remote -P!glassfish-ci-managed -Dpayara.version=${payara.version} -Dfaces.version=${faces.version} -Dsigtest.api.groupId=org.glassfish -Dsigtest.api.artifactId=jakarta.faces -Dsigtest.api.version=${sigtest.api.version} -Dpayara.arquillian.version=${payara.arquillian.version}</maven.exec.basic.options>
+        <maven.exec.goals>clean install verify</maven.exec.goals>
```

Install implies verify
```suggestion
        <maven.exec.goals>clean install</maven.exec.goals>
```

---

FISH-10299 removing verify as it is implied by install